test/governance/TimelockController.test.js

```diff
@@ -1059,11 +1087,10 @@ contract('TimelockController', function (accounts) {
         { from: proposer },
       );
       await time.increase(MINDELAY);
-      await expectRevert(
+      await expectRevert.unspecified(
```

Comment:
Is that still unspecified, or is it `FailedInnerCall` ? If unspecified, we should add a comment explaining why